.circleci/config.yml

```diff
@@ -451,30 +466,47 @@ jobs:
       - full-checkout
       - attach_workspace:
           at: .
+      - install-ghr
       - run:
           name: Release Carthage archive on GitHub
           command: |
-            GHR=ghr_v0.12.0_darwin_amd64
-            GHR_SHA256=c868ef9fc5dd8c8a397b74d84051d83693c42dd59041cb17b66f90f563477249
-            curl -sfSL --retry 5 --retry-delay 10 -O "https://github.com/tcnksm/ghr/releases/download/v0.12.0/${GHR}.zip"
-            echo "${GHR_SHA256} *${GHR}.zip" | shasum -a 256 -c -
-            unzip "${GHR}.zip"
-            ./${GHR}/ghr -replace "${CIRCLE_TAG}" MozillaAppServices.framework.zip
+            ghr -replace "${CIRCLE_TAG}" MozillaAppServices.framework.zip
   XCFramework release:
     executor: macos
     steps:
       - full-checkout
       - attach_workspace:
           at: .
+      - install-ghr
       - run:
           name: Release XCFramework archive on GitHub
           command: |
-            GHR=ghr_v0.12.0_darwin_amd64
-            GHR_SHA256=c868ef9fc5dd8c8a397b74d84051d83693c42dd59041cb17b66f90f563477249
-            curl -sfSL --retry 5 --retry-delay 10 -O "https://github.com/tcnksm/ghr/releases/download/v0.12.0/${GHR}.zip"
-            echo "${GHR_SHA256} *${GHR}.zip" | shasum -a 256 -c -
-            unzip "${GHR}.zip"
-            ./${GHR}/ghr -replace "${CIRCLE_TAG}" megazords/ios-rust/MozillaRustComponents.xcframework.zip
+            ghr -replace "${CIRCLE_TAG}" megazords/ios-rust/MozillaRustComponents.xcframework.zip
+  build-nimbus-fml:
+    executor: macos
```

Comment:
definitely not urgent nit:

I see that we use homebrew, which makes sense why we use a macos runner here! But if we can get rid of that, we can run this on a docker runner, which I think is cheaper 😄 That said, I know we'll start shipping this to iOS devs as well... and I'm not sure if once we do that we'll be forced to use macos anyways 🤷

Reply:
That was my thought, yes. Given we're already using homebrew and GHR from a macos runner, and I'm running on macos; I don't know what the M1 cross compilation story is like from linux, etc I went with the simplest option.